apps/dashboard/src/routes/dashboard/link/+page.ts

```diff
@@ -0,0 +1,19 @@
+import type { PageLoad } from './$types';
+import { error } from '@sveltejs/kit';
+
+export const load: PageLoad = async ({ parent }) => {
+    const { session, supabase } = await parent();
+
+    if (!session)
+        throw error(500, 'Failed to fetch session.');
+
+    const { data: urlData, error: err } = await supabase.from('prefix_urls').select('prefix_url').eq('created_by', session.user.id);
+    if (err)
+        throw error(500, 'Failed to fetch data.');
+
+    // TODO: Better way to figure out which URL page the user is on.
+    const currentPrefix = urlData[0].prefix_url;
```

Comment:
I need a way to figure out what prefix page the user is on when making this request, is there a way to fetch this via the session? or should I just be passing it in during the request.

Reply:
I guess it depends what "prefix page" means, may need to chat about it.